assemblies/assembly_bpmn-reference/main.adoc

```diff
@@ -11,7 +11,14 @@ include::_artifacts/author-group.adoc[]
 
 // Purpose statement for the assembly
 
-This document provides an explanation of the BPMN2 elements in the {PRODUCT} {PRODUCT_VERSION} legacy and new process designers. As a process manager, you can refer to this document for a detailed description of BPMN2 symbols and their uses when designing business or case management processes in {CENTRAL}. For more details about BPMN2, see the https://www.omg.org/spec/BPMN/2.0/About-BPMN[Business Process Model and Notation Version 2.0] specification. 
+This document provides an explanation of the BPMN2 elements in the {PRODUCT} {PRODUCT_VERSION} legacy and new process designers. As a process manager, you can refer to this document for a detailed description of BPMN2 symbols and their uses when designing business or case management processes in {CENTRAL}. For more details about BPMN2, see the https://www.omg.org/spec/BPMN/2.0/About-BPMN[Business Process Model and Notation Version 2.0] specification.
+
+[NOTE]
+====
+The legacy process designer is deprecated. It will not be updated and will be removed from {PRODUCT} in the next release.
```

Comment:
I think you should add a link to the migration topic: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-2558/#migrating-from-legacy-designer-proc